core/arch/arm/plat-imx/registers/imx-regs.h

```diff
@@ -0,0 +1,42 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2015 Freescale Semiconductor, Inc.
+ * All rights reserved.
+ * Copyright (c) 2016, Wind River Systems.
+ * All rights reserved.
+ * Copyright 2017-2018 NXP
+ *
+ */
+#ifndef PLAT_IMX_IMX_REGS_H
+#define PLAT_IMX_IMX_REGS_H
+
+#ifdef CFG_MX6
+#include <registers/imx6-regs.h>
```

Comment:
Are you sure you want to keep the `-regs` suffix? (note that your'e using `-regs` here and `_regs` elsewhere).

Reply:
the -regs are SoC level , the _regs are subsystems.
i'll keep those as is if you don't mind.

Reply:
OK, not obvious :) but fine with me.